ctapipe/utils/tests/test_CutFlow.py

```diff
@@ -0,0 +1,46 @@
+from ctapipe.utils.CutFlow import CutFlow, \
+                                  UndefinedCutException, \
+                                  PureCountingCutException
+import numpy as np
+from pytest import raises
+
+
+def test_CutFlow():
+    flow = CutFlow("TestFlow")
+    flow.set_cut("smaller5", lambda x: x < 5)
+    flow.add_cut("smaller3", lambda x: x < 3)
```

Comment:
there is none, one is alias to the other.
we talked about it where I named it `set_cut` and you mentioned maybe `add_cut` would be more suitable. but because it overwrites any existing cuts with the same name, I left `set_cut` and simply added the alias.